content/spire/try/oidc-federation-aws.md

```diff
@@ -0,0 +1,415 @@
+---
+title: AWS OIDC Authentication
+description: Using OIDC to Authenticate SPIRE to AWS S3 on Kubernetes
```

Comment:
This read a little strangely for me, since it wasn't quite clear what the subject of "on Kubernetes" was. Maybe "Using SPIRE and OIDC to authenticate workloads on Kubernetes to AWS S3"

Reply:
Agreed, done.